### PR TITLE
Remove shadow from default button

### PR DESCRIPTION
**Pull request by smyzin**
- set additional check

### CODE REVIEW

**Review by MixailE — APPROVED** [review]

File: src/components/Button.tsx [hunk 1]
```diff
@@ -55,15 +57,15 @@ const commonClasses = (
     outlineStyle('focus:outline-none'),
     opacity({ 'opacity-50': !available }),
     boxShadow({
-      'shadow-2xl': available,
-      'hover:shadow-lg': available,
-      'active:shadow-button-active': available,
+      'shadow-2xl': available && notDefaultType,
```
Comment (T-Damer) [inline src/components/Button.tsx:60]: You're doing `avaliable && notDefaultType` 3 times, should we extract it into `const`?

File: src/components/Button.tsx [hunk 2]
```diff
@@ -40,8 +40,10 @@ const commonClasses = (
...
   fullWidthOnMobile?: boolean
-) =>
-  classnames(
+) => {
+  const notDefaultType = type !== 'default'
+
+  return classnames(
     display('flex'),
     flexDirection('flex-row'),
     justifyContent(center ? 'justify-center' : undefined),
```
Comment (T-Damer) [inline src/components/Button.tsx:49]: Let's do `{}` here

**Review by T-Damer — APPROVED** [review]
Cool, cool

File: src/components/Button.tsx (hunk 1 above)
Comment (smyzin) [reply src/components/Button.tsx]: Done

File: src/components/Button.tsx (hunk 2 above)
Comment (smyzin) [reply src/components/Button.tsx]: Did

**Pull request merged**